lib/constants.js

```diff
@@ -2,8 +2,4 @@ export const SPARK_VERSION = '1.11.2'
 export const MAX_CAR_SIZE = 200 * 1024 * 1024 // 200 MB
 export const APPROX_ROUND_LENGTH_IN_MS = 20 * 60_000 // 20 minutes
 
-export const RPC_REQUEST = new Request('https://api.node.glif.io/', {
-  headers: {
-    authorization: 'Bearer Bz8B9c8hKIp+/IyePpgORexkDDq+8c9atapgGuudtQ0='
-  }
-})
```

Comment:
Thank you, @juliangruber, for testing this. I should have tested it myself to verify my assumptions. 🙈 

My understanding was based on the following docs:

https://developer.mozilla.org/en-US/docs/Web/API/Request/Request

> [options](https://developer.mozilla.org/en-US/docs/Web/API/Request/Request#options) Optional
> An object containing any custom settings that you want to apply to the request. The possible options are:
>(...)
> - [headers](https://developer.mozilla.org/en-US/docs/Web/API/Request/Request#headers)
Any headers you want to add to your request, contained within a [Headers](https://developer.mozilla.org/en-US/docs/Web/API/Headers) object or an object literal with [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) values.

I did a bit of testing in the Web Inspector console, and at least in Safari, the constructor call `new Request(RPC_REQUEST, { headers: { /* more headers */ } })` does not merge the headers from RPC_REQUEST with the headers from the options. Instead, the headers from the options replace those from the original request.